src/main/Game.java

```diff
@@ -158,4 +163,50 @@ public MouseManager getMouseManager()
         return mouseManager;
     }
 
+    public String getMENUBUTTONPATH() { return MENUBUTTONPATH; }
+
+    public boolean isInside(float x, float y, Button butt, String buttonName) {
```

Comment:
Mas ok to kung nasa Button class to.. ipapasa mo lang yung x, y ng mouse. then if inside, mag set ka ng variable sa loob ng button isClicked.. use that for logic nung changing ng state